### ISSUE TITLE
Does it support Synchronized-BatchNorm?

### ISSUE THREAD

**Issue by linshierge** [huggingface/accelerate#791]
If my model has batch normalization layer, how can I modify my code? or does it have built-in synchronized batchnorm?

**Comment by muellerzr** [comment]
Hi @linshierge, sorry for the delay in this, great question!

To use SyncBatchNorm just use the wrapper:

```python
from torch import nn

model = nn.SyncBatchNorm.convert_sync_batchnorm(model) 
accelerator.prepare(model)
```

**Comment by LittletreeZou** [comment]
When training the model containing batchnorm layer using multi-GPUs and gradient accumulations,  does SyncBatchNorm sync across micro batch size as well?
